apollo-router/src/testdata/supergraph_config.yaml

```diff
@@ -1,2 +1,2 @@
 server:
-  listen: 127.0.0.1:0
\ No newline at end of file
+  listen: 127.0.0.1:4000
```

Comment:
Strange. I wonder why that didn't cause issues before?

Reply:
Fixed in [992ddd6](https://github.com/apollographql/router/pull/1227/commits/992ddd6ae411c6fdb4b9a5d39a21132ff8a00c48)

`listen_address` functionality has been readded.